<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="paramBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="openBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getParamBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Get parameters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setParamBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity bit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud rate (bps):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Air data rate (bps):</string>
    </property>
   </widget>
   <widget class="QComboBox" name="parityBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8N1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8O1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 E1</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="baudBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="airBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0.3k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.2k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.4k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4.8k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9.6k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19.2k</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>FEC:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed mode:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>WOR timing (ms):</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fecBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Disable</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Enable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="fixedBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Disable</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Enable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="worBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>250</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>750</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1250</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1500</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1750</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IO drive mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ioBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Open-drain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Push-pull</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="modeBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wake-up</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Power-saving</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sleep</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MODE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Save changes:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="saveBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Permament</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Temporally</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
